install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -727,13 +727,14 @@ data:
 {{- if $socketLB }}
 {{- if hasKey $socketLB "enabled" }}
   bpf-lb-sock: {{ $socketLB.enabled | quote }}
-  bpf-lb-sock-terminate-pod-connections: {{ $socketLB.enabled | quote }}
 {{- end }}
 {{- if hasKey $socketLB "hostNamespaceOnly" }}
   bpf-lb-sock-hostns-only: {{ $socketLB.hostNamespaceOnly | quote }}
 {{- end }}
 {{- if hasKey $socketLB "terminatePodConnections" }}
   bpf-lb-sock-terminate-pod-connections: {{ $socketLB.terminatePodConnections | quote }}
+{{- else }}
```

Comment:
Should we replace this `else` statement here with an `else if hasKey $socketLB "enabled"` to fully preserve the previous behavior?

Reply:
Sounds good to me - added.

Reply:
Thank you!